### ISSUE TITLE
Spaces in x-data breaking named components

### ISSUE THREAD

**Issue by danharrin** [Stillat/blade-parser-typescript#62]
Alpine doesn't like it if you put a new space before and after a named component, for instance:

Before:
```
x-data="chart({
    options: {},
})"
```

After:
```
x-data="
    chart({
        options: {},
    })
"
```

This makes it freak out and break in our case. I think a good default behaviour would be to also proactively trim any whitespace from either end of the `x-data` content.

**Comment by JohnathonKoster** [comment]
This has been addressed in `1.6.3` 

I've added a new configuration option called `safeWrappingJsAttributes`, which accepts a list of regular expressions to apply the safe whitespace mode to (to help solve this in a more general way). This includes `x-data` by default, and will no longer add the leading/trailing whitespace to the attribute content.

Performing relative indentation to multi-line content in this mode had the unfortunate effect of looking pretty bad in some situations, so it will now align the content to the indentation level of the `="` on very large input 🙂

**Comment by danharrin** [comment]
Amazing, thank you @JohnathonKoster!

Hope you get that payout you deserve from Taylor and friends soon ✨
